setup.py

```diff
@@ -5,7 +5,7 @@
 
 setup(
     name='mir_eval',
-    version='0.7',
+    version='0.8',
```

Comment:
Please revert this change - version bumps should happen in a dedicated PR

Reply:
I had bumped the version number in order to allow distinguishing the locally modified version from the actual distributed v0.7 version, when running pip or poetry. The changed version then represents the `main` branch as v0.8 as an "aspirational goal", before it's a tagged release. But I've reverted the change to match the version tagging strategy used.